Examples/MAX78000/CNN/UNet-highres-demo/camera_util.c

```diff
@@ -450,7 +491,36 @@ void display_camera(void)
                     data565[j++] = rgb & 0xFF;
                 }
 
+#ifdef BOARD_EVKIT_V1
                 MXC_TFT_ShowImageCameraRGB565(0, Y_START + row, data565, w, 1);
+#else
+        tft_dma_display(0, Y_START + row, TFT_W, 1, (uint32_t *)data565);
+#endif
+
+#else //#ifndef RGB565
+
+#ifdef BOARD_EVKIT_V1
+            int j = 0;
+            for (int k = 2 * w - 1; k > 0; k -= 2) { // reverse order to display
+
+                data565[j++] = data[k + 1];
+                data565[j++] = data[k];
+            }
+#ifdef BOARD_EVKIT_V1
+            MXC_TFT_ShowImageCameraRGB565(0, Y_START + row, data565, w, 1);
+#else
+            tft_dma_display(0, Y_START + row, TFT_W, 1, (uint32_t *)data565);
+#endif
+
+#else
+#ifdef BOARD_EVKIT_V1
```

Comment:
Also possibly redundant here?  This shouldn't be defined since we're in the `#else`

Reply:
cleaned up